docs/source/development/how-to-restrict-blocks.md

```diff
@@ -22,12 +22,16 @@ The function has this signature:
     block: BlockConfigBase;
     navRoot: Content;
     contentType: string;
+    user: Object
```

Comment:
@wesleybl @davisagli The update in the docs are great but you forgot to update `@plone/types` :) 

https://github.com/plone/volto/blob/9ffe03428de664e756aa0be7016d28344a909a91/packages/types/src/config/Blocks.d.ts#L76-L83

Reply:
I will take care:

```ts
  restricted:
    | ((args: {
        properties: Content;
        block: BlockConfigBase; // TODO: This has to be extendable
        navRoot: Content;
        contentType: string;
        user: User;
      }) => boolean)
    | boolean;
```

Reply:
@sneridagh Er, I haven't really bothered to learn much about typescript yet. I kind of assumed that some linter would yell at me if the types need to be updated. Is that not right? If not, how do I know when there are type declarations that need to be updated?